client/organization.go

```diff
@@ -4,26 +4,40 @@ import (
 	"errors"
 )
 
-func (self *ApiClient) Organization() (Organization, error) {
+func (ac *ApiClient) Organization() (Organization, error) {
```

Comment:
That's the general naming convention in golang (in most cases it would even be one letter).
E.g.:
https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/terraform

But yeah, I can see how it causes a little bit of confusion.

(Also my IDE is complaining about the usage of "self").